opencog/util/Counter.h

```diff
@@ -117,30 +119,111 @@ class Counter : public std::map<T, CT>,
         return key;
     }
 
+	/* Counter operators:
```

Comment:
@rTreutlein , here and elsewhere tabs are used for indentation, while space is the dominate indentation style in this file. 

Well [smart] tabs is the dominate indentation style overall, so alternatively you could convert the rest in tabs, but would be better kept for another PR. Anyway, choose what you prefer...

Reply:
Just to be clear we're talking about smart/mixed tabs (tabs for indentation, space for alignment, see https://wiki.opencog.org/w/Development_standards#Formatting).

Taste is taste, but you might actually learn to prefer tabs, for one thing it makes moving around snappier.